src/pkgcheck/checks/metadata.py

```diff
@@ -889,6 +911,14 @@ def feed(self, pkg):
                         elif atom.slot_operator == '=':
                             yield BadDependency(
                                 attr, atom, '= slot operator used in blocker', pkg=pkg)
+                        elif not atom.blocks_strongly:
+                            weak_blocks[attr].add(atom)
+
+        for attr in ('depend', 'bdepend'):
```

Comment:
Should we add IDEPEND in this block somewhere?

Reply:
Will handle it tomorrow.
I was thinking that having a weak-blocker in `IDEPEND` was fine, but on second thought, why should it?

Reply:
It's okay if it's in there along with other deps (but unlikely), but it doesn't make sense anyway, and it's like putting it in RDEPEND. We should treat it like the "In RDEPEND but also DEPEND" case.

Reply:
Done, thanks for the clarification 😄